src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -43,25 +41,13 @@ public abstract class RazorPage : IRazorPage
         public RazorPage()
         {
             SectionWriters = new Dictionary<string, RenderAsyncDelegate>(StringComparer.OrdinalIgnoreCase);
-
             _writerScopes = new Stack<TextWriter>();
         }
 
         /// <summary>
         /// An <see cref="HttpContext"/> representing the current request execution.
         /// </summary>
-        public HttpContext Context
-        {
-            get
-            {
-                if (ViewContext == null)
-                {
-                    return null;
-                }
-
-                return ViewContext.HttpContext;
-            }
-        }
+        public HttpContext Context => ViewContext?.HttpContext;
```

Comment:
Why these unrelated style changes? Whitespace updates sure. But pure style changes usually just results in churn.